.github/workflows/publish.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         name: Create GitHub deployment
         id: deployment
         with:
-          token: "${{ github.token }}"
+          token: '${{ github.token }}'
```

Comment:
was this causing any issues or is this just a style preference?

Reply:
Good question. Automatic setting from `prettier` in the pre-commit hook. Not sure why it didn't update when I made the change initially